src/main/java/io/deephaven/csv/parsers/Parser.java

```diff
@@ -75,6 +75,10 @@ long tryParse(
             throws CsvReaderException;
 
     class GlobalContext {
+        /**
+         * The 1-based column number that the parser is working on.
```

Comment:
Is there precedent in other places for CSV columns to be 1-based?

Reply:
Yeah, I was rather conflicted about this choice.
On one hand, various entry points in `CsvSpecs` take one-based indices (like `putParserForIndex`). But one-based programming feels pretty weird elsewhere. I've changed this `colNum` thing to zero-based. Let me know if you like it better.

Reply:
I don't think it's urgent (as it's a breaking change); but https://github.com/deephaven/deephaven-csv/issues/42